### PR TITLE
Gal 254/load participating icb data

### PR DESCRIPTION
**Pull request by zainm97**
## Type of changes

- [ ] Refactoring (non-breaking change)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist

- [ ] I am familiar with the [contributing guidelines](../docs/CONTRIBUTING.md)
- [x] I have followed the code style of the project
- [x] I have added tests to cover my changes
- [ ] I have updated the documentation accordingly
- [ ] This PR is a result of pair or mob programming

---

## Sensitive Information Declaration

To ensure the utmost confidentiality and protect your and others privacy, we kindly ask you to NOT including [PII (Personal Identifiable Information) / PID (Personal Identifiable Data)](https://digital.nhs.uk/data-and-information/keeping-data-safe-and-benefitting-the-public) or any other sensitive data in this PR (Pull Request) and the codebase changes. We will remove any PR that do contain any sensitive information. We really appreciate your cooperation in this matter.

- [x] I confirm that neither PII/PID nor sensitive data are included in this PR and the codebase changes.

### CODE REVIEW

**Review by LaithM96 — DISMISSED** [review]
everything else looks good, well done :) It's important to thoroughly test your code in your specific environment to ensure it behaves as expected.

File: .github/workflows/cicd-pipeline.yaml [hunk 1]
```diff
@@ -5,6 +5,7 @@ on:
   pull_request:
     branches:
       - main
+      - develop
```
Comment (msandhuNHS) [inline .github/workflows/cicd-pipeline.yaml:8]: Need wildcard to account for every sprint etc. develop-[sprint number]

Comment (msandhuNHS) [inline .github/workflows/cicd-pipeline.yaml:8]: Need wildcard to account for every sprint etc. develop-[sprint number]

File: scripts/pipeline/seed-data.sh [hunk 2]
```diff
@@ -10,9 +10,16 @@ set -e
 # ==============================================================================
 
 function main() {
+  mkdir test-data
+
+  aws s3 cp s3://participating-icb/Participating_ICBs.csv ./test-data
+
+  source $PWD/scripts/pipeline/create-data-files.sh
+
+  echo Succefully created data
```
Comment (msandhuNHS) [inline scripts/pipeline/seed-data.sh:19]: Change to 'test data'

**Review by msandhuNHS — CHANGES_REQUESTED** [review]

**Review by LaithM96 — APPROVED** [review]
all good, thanks :)

**Review by msandhuNHS — APPROVED** [review]

**Pull request merged**